flatpak/org.nanuc.Axolotl.yml

```diff
@@ -59,16 +59,14 @@ modules:
         # Cargo offline mode
         CARGO_NET_OFFLINE: "true"
     build-commands:
-      # Install the packages from the offline cache
-      - yarn --cwd=axolotl-web install
-
       # Build axolotl-web
-      - yarn --cwd=axolotl-web run build
+      - make build-dependencies-axolotl-web
+      - make build-axolotl-web
 
       # Build axolotl
-      - cargo --offline fetch --verbose
-      - cargo --offline build --features tauri --release --verbose
-      - install -Dm755 ./target/release/axolotl -t /app/bin/
+      - cargo fetch --verbose
+      - cargo build --features tauri --release --verbose
```

Comment:
Here I would think that we could slot in `build-axolotl` without all too much hassle, perhaps we can specify the release mode and verbosity with some nice environment variables?